.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,48 @@
+name: Unit Test
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Variable Settings
+      run: |
+        pr_num=$(echo ${GITHUB_REF} | cut -d "/" -f 3)
+        echo "::set-env name=LOG_PATH::https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
+        echo "::set-env name=PR_PATH::https://github.com/${GITHUB_REPOSITORY}/pull/${pr_num}"
+
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install Third party libraries
+      run: |
+        python -V
+        sudo apt-get update
+        sudo apt install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev automake libtool lsof figlet libsecp256k1-dev jq
+
+    - name: Run MQ
+      run: |
+        docker pull rabbitmq:3.7
+        sudo docker run -d -p 5672:5672 -h localhost --name rabbit rabbitmq:3.7
+
+    - name: Install Loopchain
+      run: |
+        pip install -e .[tests]
+        grep -q "lft2.git" Makefile && pip install "git+https://github.com/icon-project/lft2.git@develop#egg=name[app]" || echo "Not 3.0 branch!"
```

Comment:
Is there any other approach?
e.g. add lft2 repo to `requirements.txt`

Reply:
I removed this line.
In `develop3.0` branch, `LFT2` is listed as one of required deps. So it can be installed at `$ pip install -e . [tests]` line.